go/host/node/node.go

```diff
@@ -305,6 +316,32 @@ func (a *Node) ReceiveTx(tx common.EncryptedTx) {
 	a.txP2PCh <- tx
 }
 
+// CreateSubscription sets up a subscription between the host and the enclave.
+func (a *Node) CreateSubscription() error {
+	id, err := uuid.NewUUID()
+	if err != nil {
+		return fmt.Errorf("could not generate new UUID for subscription")
+	}
+
+	// TODO - #453 - Link subscriptions to specific accounts.
+	subscription := common.LogSubscription{
+		Accounts: []*common.SubscriptionAccount{},
+	}
+
+	// TODO - #453 - Encrypt subscription requests, rather than just encoding them.
```

Comment:
the encrypted subscription should be passed in as a parameter from the rpc

Reply:
Added.